pyproject.toml

```diff
@@ -69,11 +69,11 @@ CMAKE_POLICY_DEFAULT_CMP0135 = "NEW"
 
 [tool.cibuildwheel]
 build-verbosity = 3
-build = ["cp39-manylinux_x86_64", "cp39-win_amd64"]  # Build for Python 3.9 manylinux x86_64 and Windows x86_64
-test-command = """pip install numpy compas pytest build && pip install {wheel} && cd {project} && python -m pytest tests/ """
```

Comment:
cibuildwheel already installs your package. And if you need extra dependencies, that's what `test-requires`, `test-extras`, and `test-groups` are for.